### ISSUE TITLE
GUI: JAWS Slit 1 synoptic macros

### ISSUE THREAD

**Issue by IsabellaRey** [ISISComputingGroup/IBEX#1617]
Descriptions of macros have incorrect examples in. Figure out what they should be and tidy up. As part of this ticket also fill in the missing JAWS descriptions or delete macros if they are not used.

**Comment by IsabellaRey** [comment]
Work for this ticket was done as part of #1289

**Comment by AdrianPotter** [comment]
Awaiting reworks for #1289

**Comment by Tom-Willemsen** [comment]
Done as part of #1289
